cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -77,9 +94,23 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 	t.Log("Jaeger-v2 is ready")
 	t.Cleanup(func() {
 		require.NoError(t, cmd.Process.Kill())
+		if t.Failed() {
+			// A Github Actions special annotation to create a foldable section
+			// in the Github runner output.
+			// https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines
+			fmt.Println("::group::Jaeger-v2 binary logs")
+			outLogs, err := os.ReadFile(outFile.Name())
+			require.NoError(t, err)
+			fmt.Printf("Jaeger-v2 output logs:\n%s", outLogs)
+
+			errLogs, err := os.ReadFile(errFile.Name())
+			require.NoError(t, err)
+			fmt.Printf("Jaeger-v2 error logs:\n%s", errLogs)
+			// End of Github Actions foldable section annotation.
+			fmt.Println("::endgroup::")
+		}
```

Comment:
perhaps it makes sense to configure codecov to ignore cmd/jaeger/internal/integration/